server/graphql/general/src/queries/tests/master_lists.rs

```diff
@@ -67,15 +67,6 @@ mod graphql {
                   name
                   code
                   description
-                  lines {
```

Comment:
Test fixes from a previous PR... should probably fix actions running out of space 👀